lib/vyasa_web/live/source_live/chapter/show_verse.ex

```diff
@@ -0,0 +1,57 @@
+defmodule VyasaWeb.SourceLive.Chapter.ShowVerse do
+  use VyasaWeb, :live_view
+  alias Vyasa.Written
+  # alias Vyasa.Corpus.Gita
+  # alias VyasaWeb.GitaLive.ImageGenerator
+  # alias Vyasa.Adapters.OgAdapter
+
+  @impl true
+  def mount(_params, _session, socket) do
+    {:ok, socket}
+  end
+
+  @impl true
+  def handle_params(%{"source_title" => source_title, "chap_no" => chap_no, "verse_no" => verse_no}, _, socket) do
+    verse = Written.get_verse_via_url_params(String.to_integer(verse_no), chap_no, source_title) |> dbg()
+
+    {:noreply,
+     socket
+     |> assign(:source_title, source_title)
+     |> assign(:chap_no, chap_no)
+     |> assign(:verse_no, String.to_integer(verse_no))
+     |> assign(:verse, verse)
+     # |> assign_meta()
+    }
+     # |> assign(:chapter, Written.get_chapter(chap_no, source_id))
+     # |> stream(:verses, Gita.verses(chap_no))
+     # |> assign(:verse, Gita.verse(chap_no, verse_no))
+     # |> assign_meta()}
+  end
```

Comment:
will refactor this when the other remaining tables are done, leaving commented-out version for now